hack/e2e-internal/e2e-status.sh

```diff
@@ -32,3 +34,10 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 prepare-e2e
 
 ${KUBECTL} version
+
+# Before running tests, ensure that all pods are 'Running'. Tests can timeout
+# and fail because the test pods don't run in time. The problem is that the pods
+# that a cluster runs on startup take too long to start running, with sequential
+# Docker pulls of large images being the culprit. These startup pods block the
+# test pods from running.
+source "${KUBE_ROOT}/hack/e2e-internal/wait-pods-running.sh"
```

Comment:
. == source

I think you should either pull it in to e2e-status, or add the boiler-plate and run it. All the scripts in this directory have that same boiler-plate variable setting (yeah, I don't like it either).